cinema/tests/test_genre_api.py

```diff
@@ -63,0 +1,61 @@
+from django.test import TestCase
+from rest_framework import status
+from rest_framework.test import APIClient
+
+from cinema.models import Genre
+
+
+class GenreApiTests(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+        Genre.objects.create(
+            name="Comedy",
+        )
+        Genre.objects.create(
+            name="Drama",
+        )
+
+    def test_get_genres(self):
+        response = self.client.get("/api/cinema/genres/")
+        genres = [genre["name"] for genre in response.data]
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(sorted(genres), ["Comedy", "Drama"])
+
+    def test_post_genres(self):
+        response = self.client.post(
+            "/api/cinema/genres/",

```

Comment:
Consider using Django's `reverse` function to generate the URL for the API endpoint. This will make the test more robust against changes in the URL patterns.